packages/router-history/src/index.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+
+import queryString from 'query-string';
+
+type HistoryType = {
+  location: {
+    pathname: string,
+    search: string
+  },
+  push: string => void
+}
+
+export default class RouterHistory {
```

Comment:
@chihsuan @abz53378 I'm thinking we should name it `historyRouter`? since It's a `router` not a `history`